### DISCUSSION TITLE
Collapsible Browser Sidebar for Icon-Only Navigation

### DISCUSSION THREAD

**Discussion by GoBooth** [lencx/Noi#140] [Ideas]
Hello [App/Dev Team],

I hope this message finds you well. I've been using Noi and have been impressed with its functionality and user interface. However, there's a feature that I believe could significantly enhance the user experience for many of us who navigate through the app regularly.

**Feature Request:**

I would like to request a collapsible "browser sidebar" that can be minimized to display only the icons, without completely disappearing. This would enable users to have a cleaner workspace while still having quick access to different sections of the app.

**Rationale:**

1. Space Efficiency: By allowing the sidebar to collapse into an icon-only view, it would provide more screen real estate for the main content area, which is particularly beneficial for users with smaller displays.
2. Quick Access: Icons are a universal language in app navigation. Having an icon-only sidebar would allow for rapid access to different IA tools within the app, like ChatGPT, Meta, Claude, etc., without the need to hover or expand for details.
3. User Preference: This feature would cater to users who prefer a minimalist interface and like to focus on the task at hand without additional textual information on the screen.

**Suggested Implementation:**

- A toggle button at the top or bottom of the sidebar to switch between expanded and icon-only views.
- Tooltips that display the name of the IA when hovering over an icon, for ease of navigation.
- Persistent user settings to remember the last used state of the sidebar (expanded or icon-only).

I believe this feature would be a welcome addition for users who appreciate a streamlined interface and would make navigation more efficient for all.

Thank you for considering my request. I am excited about the possibility of this enhancement and am happy to provide any further details if needed.

Best regards,

**Comment by lencx** [comment]
Thank you for the suggestion. If all goes well, I will implement this feature in the next version update.
